### PR TITLE
Add hero section

### PR DESCRIPTION
**Pull request by aliramazon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,6 +14,15 @@
                 <li><a href="">Facebook</a></li>
             </ul>
         </nav>
+        <header>
+            <h1>I am hero</h1>
+            <p>
```
Comment (turaninstitute) [inline index.html:19]: Hmmm. We need better text!

File: index.html [hunk 2]
```diff
@@ -14,6 +14,15 @@
...
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe
+                possimus nemo autem cupiditate eum sit quod veritatis sed in
+                veniam, cum molestias, natus nesciunt eaque delectus aperiam
+                modi totam soluta?
+            </p>
+        </header>
```
Comment (turaninstitute) [inline index.html:25]: Should we use Header here, or section is good?

Comment (aliramazon) [reply index.html:25]: Header would work better because it is not section, but Hero.

File: index.html (hunk 1 above)
Comment (turaninstitute) [reply index.html:19]: Still we need change!

**Review by turaninstitute — CHANGES_REQUESTED** [review]
Lets change the text!

File: index.html [hunk 3]
```diff
@@ -14,6 +14,10 @@
                 <li><a href="">Facebook</a></li>
             </ul>
         </nav>
+        <header>
+            <h1>I am hero</h1>
+            <p>Hero text</p>
```
Comment (turaninstitute) [inline index.html:19]: Great!

**Review by turaninstitute — APPROVED** [review]
LGTM!

**Pull request merged**